README.md

```diff
@@ -83,6 +101,12 @@ if (map.has(key)) {
 }
 ```
 
+With this proposal:
+
+```js
+map.upsert(key, old => old.doThing(), () => {})
```

Comment:
the insert function needs to be `undefined` or left out of the argument list, otherwise `undefined` gets inserted into the map.

Reply:
I miswrote the fix, should have been `map.upsert(key, old => old.doThing())` without the extra `.doThing()`